gradle/java-module.gradle

```diff
@@ -112,17 +112,72 @@ dependencies {
 
 tasks.withType( JavaCompile ) {
 	options.encoding = 'UTF-8'
-	sourceCompatibility = project.baselineJavaVersion
```

Comment:
just wondering, was it necessary to remove these?

Reply:
I didn't remove these, I just moved them further below. We can't use `sourceCompatibility` and such when toolchains are enabled, so I needed an `if` block.